assets/css/popup.css

```
@@ -0,0 +1,76 @@
+body {
+    width: 200px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: stretch;
+    background-color: #f6f8fa;
+    padding-bottom: 0.5rem;
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif;
+    text-align: center;
+}
+
+h1 {
+    color: #24292e;
+    margin-bottom: 1rem;
+    margin-top: 0.5rem;
+}
+
+.btn {
+    cursor: pointer;
+    border-color: #1f232826;
+    color: white;
+    font-size: .875rem;
+    font-weight: 500;
+    border-width: 1px;
+    border-style: solid;
+    border-radius: 6px;
+    padding: 0.5rem 0.75rem;
+    transition: background 0.2s;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 0.1rem;
+}
+.btn svg {
```

```suggestion
}

.btn svg {
```

Just add a blank line between all classes and it should be fine.